### DISCUSSION TITLE
Great work! May I ask if you have considered adding the bbox of the element executed by the action on the image to the data set, which is quite useful for the CV based methods.

### DISCUSSION THREAD

**Discussion by XuRui314** [McGill-NLP/weblinx#15] [Q&A] [answered]
Like the red bbox here:
![image](https://github.com/McGill-NLP/weblinx/assets/80871590/f993d0ad-7c9e-4a18-89b6-de695baf7890)

**Comment by xhluca** [comment, marked as answer]
Yes the bboxes/bbox-*.json are part of the dataset (see `weblinx-full` on huggingface) which map an element id to coordinates. the target element id can be found in the `metadata.json`. If you want a tutorial you can check out the `modeling/llama/eval.py` or the new colab notebook: https://colab.research.google.com/github/McGill-NLP/weblinx/blob/main/examples/WebLINX_Colab_Notebook.ipynb

**Reply by xhluca** [reply]
Btw I've converted this to discussions rather than issues

**Reply by XuRui314** [reply]
Thx for sharing :)
